nano/node/unchecked_map.hpp

```diff
@@ -62,11 +62,10 @@ class unchecked_map
 	nano::condition_variable condition;
 	nano::mutex mutex;
 	std::thread thread;
+	unsigned max_unchecked_blocks;
```

Comment:
You could also use const here but only if we will not allow changing this dynamically.

Reply:
Fixed